### PR TITLE
Support xsmall breakpoint for site layout and examples

### PR DESCRIPTION
**Pull request by taysea**
[Deploy Preview](https://deploy-preview-2371--keen-mayer-a86c8b.netlify.app/)

#### What does this PR do?

We are moving towards supporting an `xsmall` breakpoint. This is a breaking change in the theme and requires instances where people are using `small` as a breakpoint to shift to mobile designs to change to check for `xsmall` and `small`. This PR contains the changes needed for the DS site.

#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [ ] Small, medium, and large screen sizes
- [ ] Cross-browsers (FireFox, Chrome, and Safari)
- [ ] Light & dark modes
- [ ] All hyperlinks route properly

**Accessibility Checks**
- [ ] Keyboard interactions
- [ ] Screen reader experience
- [ ] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [ ] Console is free of warnings and errors
- [ ] Passes E2E commit checks
- [ ] Visual snapshots are reasonable

#### What are the relevant issues?
Related to https://github.com/grommet/hpe-design-system/issues/1794
Related to https://hpe.slack.com/archives/C04LMJWQT/p1645007683789829

#### Should this PR be mentioned in Design System updates?
No.

#### Is this change backwards compatible or is it a breaking change?
Yes.

### CODE REVIEW

File: aries-site/src/examples/templates/content-layouts/ResponsiveContentLayoutExample.js [hunk 1]
```diff
@@ -79,7 +79,9 @@ const PageContent = () => {
       <Grid gap={parentGrid.gap[size]} columns={parentGrid.columns[size]}>
         {/* Content Block 1 is top priority content. At narrow breakpoints, 
         place as first content element. Otherwise, place in second column. */}
-        {(size === 'small' || size === 'xsmall') && <ContentBlock title="1" />}
+        {(['xsmall', 'small'].includes(size) || size === 'xsmall') && (
```
Comment (halocline) [inline aries-site/src/examples/templates/content-layouts/ResponsiveContentLayoutExample.js:82]: I think the size === 'xsmall' is redundant now.

```suggestion
        {['xsmall', 'small'].includes(size) && (
```

File: aries-site/src/examples/templates/dashboards/TwoColumnDashboard.js [hunk 2]
```diff
@@ -92,7 +92,9 @@ const PageContent = () => {
       <Grid columns={parentGrid.columns[size]} gap={parentGrid.gap[size]}>
         {/* RecentActivity is top priority content. At narrow breakpoints, 
         place as first content element. Otherwise, place in second column. */}
-        {(size === 'small' || size === 'xsmall') && <RecentActivity />}
+        {(['xsmall', 'small'].includes(size) || size === 'xsmall') && (
```
Comment (halocline) [inline aries-site/src/examples/templates/dashboards/TwoColumnDashboard.js:95]: Redundant?

```suggestion
        {['xsmall', 'small'].includes(size) && (
```

File: aries-site/src/examples/templates/dashboards/TwoColumnDashboard.js [hunk 3]
```diff
@@ -122,7 +124,7 @@ const PageContent = () => {
             </Grid>
           </Box>
         </Box>
-        {size !== 'small' && size !== 'xsmall' && (
+        {!['xsmall', 'small'].includes(size) && size !== 'xsmall' && (
```
Comment (halocline) [inline aries-site/src/examples/templates/dashboards/TwoColumnDashboard.js:127]:
```suggestion
        {!['xsmall', 'small'].includes(size) && (
```

File: aries-site/src/examples/components/tag/CreateTag/TagPageHeader.js [hunk 4]
```diff
@@ -1,22 +1,27 @@
...
+import { PageContainerContext } from '../../..';
 
-export const TagPageHeader = ({ description }) => (
-  <Box gap="medium" flex={false}>
-    <Box gap="xsmall" flex={false}>
-      <Heading level={2} margin="none">
-        Create and Assign Tags
-      </Heading>
+export const TagPageHeader = ({ description }) => {
+  const { ...pageContainer } = useContext(PageContainerContext);
```
Comment (halocline) [inline aries-site/src/examples/components/tag/CreateTag/TagPageHeader.js:7]: I this change related? I'm not immediately seeing the connection.

Comment (taysea) [reply aries-site/src/examples/components/tag/CreateTag/TagPageHeader.js:7]: Thanks for catching. It is not relevant. I will remove.

**Review by halocline — APPROVED** [review]

**Pull request merged**